locales/de.json

```diff
@@ -2980,7 +2980,7 @@
 			},
 			"text": {
 				"shareWithOthersTeachers": "mit anderen Lehrern teilen",
-				"warningMain": "Hinweis: Beim Klicken des Links verlassen Sie das Angebot der Schul-Cloud",
+				"warningMain": "Hinweis: Beim Klicken des Links verlassen Sie das Angebot der Schul-Cloud Brandenburg.",
 				"warningFooter": "Die Nutzung dieser Angebote unterliegt gegebenenfalls anderen rechtlichen Bedingungen. Bitte schauen Sie daher in die Datenschutzerklärung des externen Anbieters!"
```

Comment:
normally, this should be Brandenburg Schul-Cloud and should be dynamic, from short_title

"warningMain": "Hinweis: Beim Klicken des Links verlassen Sie das Angebot der {{shortTitle}}",

<h5>{{$t "topic._topic.text.warningMain" (dict "shortTitle" @root.theme.short_title)}}</h5>

Reply:
The error message is only for Brandenburg so it doesn't matter. The other instances doesn't need this error message. I see your point, but the whole thing is going to be changed when the modal replaces this text warning